### ISSUE TITLE
Rule constructor missing `selector` property

### ISSUE THREAD

**Issue by pascalduez** [postcss/postcss#525]
Greetings,

I'm trying to upgrade a plugin to PostCSS 5, and having an issue while creating a new rule through the `Rule`  constructor.

```js
let selectors = [  '...'  ];

let newRule = postcss.rule({
  selectors,
  nodes: atRule.nodes,
  source: atRule.source,
  semicolon: true,
});
``` 

Leads to:
```
TypeError: Cannot read property 'match' of undefined
    at Rule.set (/[...]/node_modules/postcss/lib/rule.js:42:38)
``` 

The `Rule.selectors`  setter is somehow relying on [`this.selector`](https://github.com/postcss/postcss/blob/master/lib/rule.es6#L19) which is not declared in the class nor in its parents.

So is it still possible to instantiate a new rule with a `selectors` Array? I don't see any test nor doc for it.
Where should `selector` be located. `this.selector` or `this.raws.selector` ?

Bug or I am just missing something obvious ?

**Comment by ai** [comment]
Fixed f8b6802

**Comment by ai** [comment]
Released in 5.0.3.
